charts/generic/pod-io-stress/experiment.yaml

```diff
@@ -93,7 +93,7 @@ spec:
 
       # provide lib image
       - name: LIB_IMAGE
-        value: 'gaiaadm/pumba'
+        value: 'litmuschaos/go-runner:ci'
```

Comment:
do we want to use ci tag or latest ? (if latter, we should merge this once 1.12 is pushed)

Reply:
ok, e2e was running with the older image that is `gaiaadm/pumba` and failing. I'll try to give custom `LIB_IMAGE` and run. 
Thanks!